src/wallet/rpcdump.cpp

```diff
@@ -1284,7 +1284,8 @@ static UniValue ProcessImport(CWallet * const pwallet, const UniValue& data, con
                 }
 
                 // add to address book or update label
```

Comment:
this comment is supposed to be updated
```suggestion
                // if not internal add to address book or update label
```